### ISSUE TITLE
documentation sucks

### ISSUE THREAD

**Issue by defvol** [rodowi/unmapped#1]

**Comment by bkowshik** [comment]
Yes @rodowi would be great to have some documentation. Let me know how I can help.

**Comment by defvol** [comment]
lol,
let's start with an AMA
what's your first question?

**Comment by bkowshik** [comment]
Ha ha! Really excited to have the complete working setup.

> what's your first question?

I am not a 💯 but, is there a `Dockerfile` missing from the repository?

**Comment by defvol** [comment]
yea it's a good idea to upload a custom Dockerfile, but in the meantime you can pull the official image
take a look at https://github.com/tensorflow/tensorflow/blob/8cac382a5425d64f3083cb5adec525baa163e18e/tensorflow/tools/docker/README.md#L6, hope that helps
